.travis.yml

```diff
@@ -12,5 +13,20 @@ before_install:
   - npm install -g grunt-cli
   - npm config set strict-ssl false
 install: npm install
-script:
-  - npm run test
+
+matrix:
+  # The acceptance tests only work with node 4 or higher.
+  exclude:
+  # Remove the original job for node 4.4.3, without the acceptance tests.
+  - node_js: "4.4.3"
+  include:
+  # Include an updated job, with the acceptance tests. This way we're not
+  # running the 4.4.3 build twice with one build including acceptance tests.
+  - node_js: "4.4.3"
+    script:
+      - npm link
+      - git clone https://github.com/feedhenry/sync-acceptance-testing.git
+      - cd sync-acceptance-testing
+      - npm install
+      - npm link fh-mbaas-api
+      - npm test
```

Comment:
Fixed to run unit & integration tests for node 0.10 & 4.

Acceptance tests are run for node 4 as another matrix job